README.md

```diff
@@ -8,125 +8,98 @@ Please use the [covid](https://github.com/epidemics/covid/issues/new/choose) rep
-   `./do generate-gleam-batch -D 2020-04-15 -c JK default.xml estimates-2020-04-15.csv`
-   The batch file now contains all the scenario definitions and initial populations.
-   Note the estimate input specification may change.
-
-4. Export Gleam simulation XML files in Gleamviz (not while gleamviz is running!).
-   `./do export-gleam-batch out/batch-2020-04-16T03:54:52.910001+00:00.hdf5`
-
-5. Start gleamviz. You should see the new simulations loaded. Run all of them and "Retrieve results" (do not export manually). Exit gleamviz.
-
-6. ./do import-gleam-batch out/batch-2020-04-16T03:54:52.910001+00:00.hdf5
-    will output to  out/batch-out.hdf5
-   (Gleamviz must be stopped before that.) After this succeeds, you may delete the simulations from gleamviz.
-
-7. Generate web export (additional data are fetched from [config.yml](https://github.com/epidemics/epimodel/blob/master/config.yaml#L16))
+We are using [luigi](https://luigi.readthedocs.io/en/stable/index.html) as the workflow framework. This
+readme doesn't include description of how to use `luigi` so please refer to the project documentation
+to understand how to tweak this.
 
-   `./do web-export out/batch-out.hdf5 data/sources/estimates-JK-2020-04-15.csv`
-
-8. Export the generated folder to web! Optionally, set a channel for testing first.
-   `./do web-upload -d out/export-2020-04-03T02:03:28.991629+00:00 ttest28`
-
-Alternatively, steps 7 and 8 can be combined:
-
-`./do web-export out/batch-2020-04-16T03:54:52.910001+00:00.hdf5 data/sources/estimates-JK-2020-04-15.csv --upload ttest28`
-
-### Running pipeline with workflow commands
-
-An alternative way to run the pipeline is as follows:
+## Running the pipeline
+### Example with faked data
+This example skips the `UpdateForetold` and `ExtractSimulationsResults` task by providing their output.
+In reality, you want to actually run gleamviz in between and provide Foretold channel to get data via API.
+This by default uses data in `data/inputs` and exports data to `data/outputs/example`.
+```
+# `poetry shell`  # if you haven't done already
+./luigi WebExport \
+    --export-name test-export \
+    --UpdateForetold--output-directory data-dir/inputs/fixtures \
+    --ExtractSimulationsResults--output-directory data-dir/inputs/fixtures \
+    --ExtractSimulationsResults-single-result this-is-now-ignored
+```
 
-1. Update Johns Hopkins and Foretold data, generate batch file from estimates and basic Gleam XML definition and export Gleam simulation XML files to Gleamviz (not while gleamviz is running!):
-   `./do workflow-prepare-gleam -D 2020-04-15 -c JK default.xml estimates-2020-04-15.csv`
+After the pipeline finishes, you should see the results in `data-dir/outputs/example/`
 
-2. Start gleamviz. You should see the new simulations loaded. Run all of them and "Retrieve results" (do not export manually). Exit gleamviz.
+### The usual flow
+You provide all file inputs, foretold_channel and parameters, tweak configs to your liking and then:
```

Comment:
This is assuming a lot of knowledge on the user’s part. Maybe this is explained later, but it’s not obvious to me where to look